docs/getting-started/release-notes/v2_9_0.md

```diff
@@ -18,31 +18,91 @@ Zowe Version 2.9.0 contains the enhancements that are described in the following
+
+- Added the `zowe files copy dsclp` command to copy a data set from one LPAR to another, making data transfer between LPARs easier. ([#1098](https://github.com/zowe/zowe-cli/issues/1098))
+
+- Re-enabled color in the daemon client to differentiate text displayed in the terminal. ([#1379](https://github.com/zowe/zowe-cli/issues/1379))
+
 ### Zowe Explorer
 
+- Added option to save unique data set attributes as a template after allocation for future use. ([#1425](https://github.com/zowe/vscode-extension-for-zowe/issues/1425))
+
+- Added "Cancel Job" feature for job nodes in Jobs panel view. ([#2251](https://github.com/zowe/vscode-extension-for-zowe/issues/2251))
+
+- Enhanced ID generation for parent tree nodes to ensure uniqueness. ([#2325](https://github.com/zowe/vscode-extension-for-zowe/pull/2325))
+
+- Added support for custom credential manager extensions. ([#2212](https://github.com/zowe/vscode-extension-for-zowe/issues/2212))
 ## Bug fixes
 
 Zowe Version 2.9.0 contains the bug fixes that are described in the following topics.
 
-### Zowe installation and packaging
-
 ### Zowe Application Framework
 #### ZLux App Server
 * Recognizers from multiple plug-ins could not be merged due to an error in the merge code execution at startup. ([#256](https://github.com/zowe/zlux-app-server/pull/256))
 
 ### Zowe API Mediation Layer
 
+### Imperative CLI Framework
+
+- Added logic to display a warning in cases where a `null` command handler is provided. ([#990](https://github.com/zowe/imperative/pull/990))
+
+- Fixed a logic error in the `config list` command that caused unwanted behavior when a positional argument and the `--locations` option were both passed in the command. ([#989](https://github.com/zowe/imperative/pull/989))
+
+- Reduced file loading time by searching for command definitions with the `fast-glob` NPM module instead of the `glob` module. ([#](https://github.com/zowe/imperative/pull/986))
+
+- Removed validation of the deprecated `pluginHealthCheck` property because it was an unused feature. ([#980](https://github.com/zowe/imperative/issues/980))
+
 ### Zowe CLI
 
+- Enabled ANSI in Windows-based terminals to better display progress bars when using daemon mode. ([#1701](https://github.com/zowe/zowe-cli/issues/1701))
+
+- Changed daemon to spawn as its own process to avoid unintentional termination of the daemon mode. ([#1241](https://github.com/zowe/zowe-cli/issues/1241), [#1277](https://github.com/zowe/zowe-cli/issues/1277), [#1309](https://github.com/zowe/zowe-cli/issues/1309))
+
+- Fixed `--secondary` data set allocation option being specified as `1` unit on `BLANK` type data sets with the `zowe files create data-set` command. ([#1595](https://github.com/zowe/zowe-cli/issues/1595))
+
+- Fixed `--range` option so it is not ignored on the `zowe files view uss-file` command. ([#1717](https://github.com/zowe/zowe-cli/pull/1717))
```

Comment:
Should we update the list items so that there aren't any new lines between each list item?

e.g.:

```
- ex 1
- ex 2
- ex 3
```

vs.

```
- ex 1

- ex 2

- ex 3
```